internal/plugins/ansible/v1/scaffolds/internal/templates/dockerfile.go

```diff
@@ -43,11 +44,14 @@ func (f *Dockerfile) SetTemplateDefaults() error {
 	f.TemplateBody = dockerfileTemplate
 	f.RolesDir = constants.RolesDir
 	f.PlaybooksDir = constants.PlaybooksDir
-	f.ImageTag = strings.TrimSuffix(version.Version, "+git")
+
+	if f.AnsibleOperatorVersion == "" {
+		return errors.New("docker file scaffold: ansible-operator version must be set")
```

Comment:
Is this an error that a user would be able to resolve? If not, should this panic instead of returning an error?

Reply:
Given the way these scaffold fields are used (internally, no user input, not dependent on environment) these should always be caught by tests. Panic is fine with me.

Reply:
Done. Sorry for the delay.